Format/TextFormatter.cs

```diff
@@ -239,7 +239,13 @@ protected static string DefaultProcessImagesDelegate(TextFormatter formatter, Ma
 		/// <returns>Formatted image value</returns>
 		public virtual string ProcessImages(Match image)
 		{
-			return $"<img border='0' src='{image.Groups["url"].Value.EncodeAgainstXSS()}' />";
+			var src = image.Groups["url"].Value.EncodeAgainstXSS();
+			var decorator = image.Groups["decorator"].Value;
+
+			if (!string.IsNullOrEmpty(decorator))

```

Comment:
It is weird to insert a string without checks. At the very least, this is a danger of XSS. It is necessary to check for a limited set of variants.

Reply:
@andrewvk Yes, did it from the beginning (i.e. inserted `small|large`); but the pattern is `\w+` allows only plain sequential alphabetical. You can't do any evil with this? And the idea is maybe in the future you would allow something else then recompilation is not needed.  Or just replace with `small|large`?

Reply:
@andrewvk ping

Reply:
@andrewvk ping. any update?

Reply:
@andrewvk ping... 😿

Reply:
The typing error should definitely be ignored by simply not generating the class at all. And custom css makes no sense on a public site. So I would still add a check for a limited set of options.

Reply:
@andrewvk limited to large|small, please check when you have time.